images/linux/scripts/installers/azpowershell.sh

```diff
@@ -65,3 +71,4 @@ DocumentInstalledItem "Az Module (2.6.0)"
 DocumentInstalledItem "Az Module (2.8.0)"
 DocumentInstalledItem "Az Module (3.1.0)"

```

Comment:
@maxim-lobanov as an aside, we should probably get rid of 3.1 and 3.5? Support only one revision per major version.

Reply:
I agree but I think it is breaking change for customers because they point directly to 3.1 for example in their pipelines